airflow/contrib/operators/dataproc_operator.py

```diff
@@ -157,6 +157,22 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
 
+class DataProcSparkROperator(DataprocSubmitSparkRJobOperator):
```

Comment:
It is not necessary. This is only needed if this operator was available in Airflow 1.10.x

Reply:
merci, will remove